src/GafferImage/CDL.cpp

```diff
@@ -131,6 +131,8 @@ bool CDL::affectsTransform( const Gaffer::Plug *input ) const
 
 void CDL::hashTransform( const Gaffer::Context *context, IECore::MurmurHash &h ) const
 {
+	ImagePlug::GlobalContext c( context );
```

Comment:
Can the responsibility for this be pushed up onto the OpenColorIOTransform class? That seems preferable to having to remember it in all the base classes...